### PR TITLE
[ISSUE 7265] Doc: Adding how to debug in Idea document

### LINKED ISSUE #7265: [Enhancement] Add document for Debugging RocketMQ in Idea
### Before Creating the Enhancement Request

- [X] I have confirmed that this should be classified as an enhancement rather than a bug/feature.

### Summary

So far, it seems that there is no document for developers to start and debug rocketmq in Idea, which is necessary for contributors or anyone who wants to read source code. 

### Motivation

The idea is very common in Java programming, adding a document for debugging rocketmq locally is necessary.

### Describe the Solution You'd Like

Add a document of steps to debug in Idea

### Describe Alternatives You've Considered

null

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by joeCarf**
### Which Issue(s) This PR Fixes

Fixes #7265 

### Brief Description
Adding how to debug rocketmq in Idea document

### How Did You Test This Change?
no need to test

### CODE REVIEW

**Comment by joeCarf** [comment]
@RongtongJin @ShannonDing Hi, could you please review this pr? Thanks a lot :)

**Review by lizhimins — COMMENTED** [review]
也来个英语版？

**Comment by joeCarf** [comment]
> 也来个英语版？

Good idea! I will work on it these days

**Comment by joeCarf** [comment]
> 也来个英语版？

done!

File: docs/en/Debug_In_Idea.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+The broker[broker-a,192.169.1.2:10911] boot success...
+```
+
+### Step3: Send or Consume Messages
+RocketMQ startup is now complete. You can use the examples provided in `/example` to send and consume messages.
+
+### Additional: Start the Proxy locally.
+1. RocketMQ 5.x introduced the Proxy mode. Using the `LOCAL` mode eliminates the need for `Step2`. The startup class is located at `org.apache.rocketmq.proxy.ProxyStartup`.
+2. Add the runtime parameter `ROCKETMQ_HOME=<rocketmq repository directory>` in `Idea-Edit Configurations`.
+3. Create a new configuration file named `rmq-proxy.json` in the `/conf/` directory.
```
Comment (Diwakar-Gupta) [inline docs/en/Debug_In_Idea.md:44]: In local mode we are running both proxy and broker so we also need to add `conf/broker.conf` ?

Comment (joeCarf) [reply docs/en/Debug_In_Idea.md:44]: 截屏2023-10-07 12 54 37
in proxy mode, if not set, the broker conf path will be set as default

Comment (joeCarf) [reply docs/en/Debug_In_Idea.md:44]: so we also need it

**Comment by joeCarf** [comment]
@xdkxlk hi, could you please review this pr :)

**Comment by joeCarf** [comment]
ping @lizhimins @RongtongJin

**Review by xdkxlk — APPROVED** [review]

**Pull request merged**